src/test/java/com/google/api/generator/gapic/composer/FileDiffInfraDummyTest.java

```diff
@@ -0,0 +1,94 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.api.generator.gapic.composer;
+
+import static junit.framework.Assert.assertEquals;
+
+import com.google.api.generator.engine.ast.ClassDefinition;
+import com.google.api.generator.engine.ast.CommentStatement;
+import com.google.api.generator.engine.ast.LineComment;
+import com.google.api.generator.engine.ast.ScopeNode;
+import com.google.api.generator.engine.writer.JavaWriterVisitor;
+import com.google.api.generator.gapic.model.GapicClass;
+import com.google.api.generator.gapic.model.GapicClass.Kind;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.Test;
+
+public class FileDiffInfraDummyTest {
```

Comment:
Could we please add a TODO to remove this afterwards, so we don't forget?

Reply:
yes, I have a TODO below for myself in the comment, but please let me know if I'm understanding this wrong. 

// TODO(xiaozhenliu): remove this test class once the file-diff infra is in place and well-tested.